src/eduid_userdb/userdb.py

```diff
@@ -288,3 +288,19 @@ def update_user(self, obj_id, attributes):
                         '$set': attributes,
                     }
                 )
+
+    def get_identity_proofing(self, user):
+        """
+        Return the proofing urn value
+
+        :param user: The user object
+        :type user: User
+        """
+        al1_urn = 'http://www.swamid.se/policy/assurance/al1'
+        al2_urn = 'http://www.swamid.se/policy/assurance/al2'
+        user = self.get_user_by_id(user.user_id)
+        if user is not None:
+            if user.nins.count > 0:
+                return al2_urn
```

Comment:
user.nins can be both verified and unverified now, right? so this must be

if user.nins.verified.count > 0: